up-linux-streamer-plugin/Cargo.toml

```diff
@@ -48,7 +48,7 @@ up-transport-zenoh = { version = "0.3.0"  }
 up-transport-vsomeip = { git = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust.git", tag = "v0.3.0", default-features = false }
 up-streamer = { path = "../up-streamer" }
 usubscription-static-file = {path = "../utils/usubscription-static-file"}
-zenoh = { version = "1.0.0", features = ["unstable", "internal", "plugins"] }
+zenoh = { version = "1.0.0", features = ["default", "plugins"] }
```

Comment:
how about defining the Zenoh dependency at the workspace level?

Reply:
Yeah, after doing this yesterday and your comment, I think that's what I'll try to do. I'm going to try to define some of the transport specific stuff at the workspace-level (including Zenoh), make them optional and then again optionally include them based on the same feature flag. Let's see how that goes.

Reply:
Hmm, seems like it's not possible to define optional dependencies at the workspace level. I'm looking to go the other direction. Lemme try some stuff.

Reply:
Could you take another look at where I ended up?